akka-remote/src/main/scala/akka/remote/artery/TaskRunner.scala

```diff
@@ -101,7 +101,7 @@ private[akka] object TaskRunner {
       val yielding = 5 * idleCpuLevel
       val minParkNanos = 1
       // park between 250 and 10 micros depending on idleCpuLevel
-      val maxParkNanos = MICROSECONDS.toNanos(280 - 30 * idleCpuLevel)
+      val maxParkNanos = MICROSECONDS.toNanos(250 - (80 * (idleCpuLevel - 1) / 3))
```

Comment:
1 and 10 is actually used with SleepingIdleStrategy and BusySpinIdleStrategy above. So no need to change this calculation here AFAICS.